code/__defines/admin.dm

```diff
@@ -22,21 +22,13 @@
 #define ROUNDSTART_LOGOUT_REPORT_TIME 6000 // Amount of time (in deciseconds) after the rounds starts, that the player disconnect report is issued.
 
 // Admin permissions.
-#define R_BUILDMODE     0x1
-#define R_ADMIN         0x2
-#define R_BAN           0x4
-#define R_FUN           0x8
-#define R_SERVER        0x10
-#define R_DEBUG         0x20
-#define R_POSSESS       0x40
-#define R_PERMISSIONS   0x80
-#define R_STEALTH       0x100
-#define R_REJUVINATE    0x200
-#define R_VAREDIT       0x400
-#define R_SOUNDS        0x800
-#define R_SPAWN         0x1000
-#define R_MOD           0x2000
-#define R_MENTOR        0x4000
-#define R_HOST          0x8000 //higher than this will overflow
+#define R_FUN           0x1
+#define R_SERVER        0x2
+#define R_DEBUG         0x4
+#define R_PERMISSIONS   0x8
+#define R_MENTOR        0x10
+#define R_MOD           0x20
+#define R_ADMIN         0x40
+#define R_HOST          0x80
```

Comment:
R_HOST only allow delete admin memo created by other admins and used as default required right for admin secret but overrided in each case.

Reply:
So should we get rid of this flag?

Reply:
Think so. We had remove `R_BUILDMODE` because of same reason hadn't we?